src/containers/TermsOfUse/index.tsx

```diff
@@ -0,0 +1,223 @@
+import React, {FC} from 'react';
+
+import {Container} from 'components';
+
+import './TermsOfUse.scss';
+
+const TermsOfUse: FC = () => {
+  return (
+    <Container className="TermsOfUse" maxWidth={720}>
+      <p className="TermsOfUse__last-updated">Last Updated: June 22, 2021</p>
+      <h1 className="TermsOfUse__heading">Terms of Use</h1>
+      <div className="TermsOfUse__main-text">
+        <p className="TermsOfUse__text">
+          This website (the “Site”) is operated by TheNewBoston (“we”, “us”, “our”). To contact us, please email
+          contact@thenewboston.com.
+        </p>
+        <p className="TermsOfUse__text">
+          By using our Site, you confirm that you accept these Terms of Use (“Terms”) and you agree to comply with them.
+          If you do not agree to these Terms, you must not use our Site.
+        </p>
+      </div>
+      <section className="TermsOfUse__section">
+        <h2 className="TermsOfUse__section-heading">OTHER TERMS THAT MAY APPLY</h2>
+        <p className="TermsOfUse__text">
+          These Terms of Use refer to the following additional terms that also apply to your use of our Site:
+        </p>
+        <ul className="TermsOfUse__list">
+          <li className="TermsOfUse__list-item">
+            <span className="TermsOfUse__text-highlight">Privacy Policy</span>, which sets out how we process personal
+            data we receive from you. If you do not wish your personal data to be processed in this way, you must not
+            use this Site.
+          </li>
+        </ul>
+        <p className="TermsOfUse__text">
+          In some instances, additional terms and conditions, such as a software or product license, may apply to a
+          service or product offered through the Site (“Additional Terms”). To the extent there is a conflict between
+          these Terms and any Additional Terms, the Additional Terms will control with respect to such service or
+          product unless the Additional Terms expressly state otherwise.
+        </p>
+      </section>
+      <section className="TermsOfUse__section">
+        <h2 className="TermsOfUse__section-heading">CHANGES TO THESE TERMS AND OUR SIDE</h2>
+        <p className="TermsOfUse__text">
+          We amend these Terms from time to time. It is your responsibility to be informed of amendments in these Terms.
+          We will provide you with the opportunity to review changes before they become effective by indicating the
+          dates they were last updated (“Last updated:”) and the date that they will become effective (“Effective:”).
```

Comment:
hi @led76marx @ma2b0043 seems like we will need the `Effective:` date section on top of the documents, but currently in our design as well as our implementation, this is not included.

Reply:
do you want a new design for this, or should I ask kristy to make adjustment to current one?

Reply:
@ma2b0043 Updating the current one will do. I'll make changes according to that.

Reply:
I'll ask Mr K to update the designs, but he cant, we will have to wait for a week because kristy is on vacation. Can we just go with this design(if Mr K cant make changes to this) and change it when Kristy comes back?

Reply:
yeah sure we will go with the current design for now